File: semester I/Praktikum Pemrograman I/Modul4_JovanGilbertNatamasindah_2310817310002/Python/PRAK402-2310817310002-JovanGilbertNatamasindah.py
ANGKA = int(input("Masukkan Angka : "))

for (GANJIL) in range(1,ANGKA + 1) :
    
    if (GANJIL % 2 != 0) :
        print(GANJIL, end=" ")

print()

for (GENAP) in range(ANGKA,1,-1) :
    
    if (GENAP % 2 == 0) :
        print(GENAP, end=" ")


# penjelasan dari penggunaan -1 = https://sparkbyexamples.com/python/python-for-loop-in-backwards/#:~:text=You%20can%20use%20range(),values%20from%20the%20last%20index.

File: semester I/Praktikum Pemrograman I/Modul4_JovanGilbertNatamasindah_2310817310002/C/PRAK402-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>

void main() {
    int ANGKA;

    printf("Masukkan Angka : ");
    scanf("%d", &ANGKA);

    for( int GANJIL = 1 ; GANJIL <= ANGKA ; GANJIL++) {
        
        if( GANJIL % 2 != 0) {
            printf("%d ", GANJIL);
        }

    }    

    printf("\n");

    for (int GENAP = ANGKA ; GENAP >= 1 ; GENAP--) {

         if (GENAP % 2 == 0) {
            printf("%d ", GENAP);
        }
        
    }
}